crates/build/src/args.rs

```diff
@@ -45,7 +45,9 @@ impl TryFrom<&VerbosityFlags> for Verbosity {
 }
 
 /// Denotes if output should be printed to stdout.
-#[derive(Clone, Copy, Default, serde::Serialize, serde::Deserialize, Eq, PartialEq)]
+#[derive(
+    Clone, Copy, Default, serde::Serialize, serde::Deserialize, Eq, PartialEq, Debug,
+)]
```

Comment:
```suggestion
#[derive(Clone, Copy, Default, serde::Serialize, serde::Deserialize, Eq, PartialEq)]
```

Probably a leftover from you developing.